www/templates/html/Media.tpl.php

```diff
@@ -128,7 +130,17 @@
                         
                     </div>  
                 </div>          
-
+                <hr>
+                
+                <?php if($getTracks){
+                    echo '<div class="mediahub-onpage-captions">';
+                    echo $savvy->render($context, 'MediaPlayer/Transcript.tpl.php');
+                    echo '</div>';
+                    }
+                ?>
+               
+                <hr>
+                <h6 class="wdn-sans-serif">Description</h6>
```

Comment:
Whatever is logical. If it is the only heading element after the h1, it should probably be an h2.